react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx

```diff
@@ -644,11 +644,16 @@ const useStyles = makeStyles({
         position: "fixed",
     },
 });
-export const MapInContainer = () => {
+
+export const MapInContainer = (args) => {
     const classes = useStyles();
     return (
         <div className={classes.main}>
-            <DeckGLMap {...exampleData[0]} />
+            <DeckGLMap {...args} />
         </div>
     );
 };
+
+MapInContainer.args = {
+    ...exampleData[0],
```

Comment:
I think we should avoid using the exampleData, as it is a specific example of using a json file to create the map props, which most consumers probably won't do.